src/app/helpers/announcement/controller.js

```diff
@@ -166,14 +167,15 @@ class AnnouncementController {
           'details': {
             'title': data.title,
             'type': data.type,
-            'description': data.description
+            'description': data.description,
+            'from':data.from,
           },
           'target': JSON.stringify(data.target),
           'links': data.links
         }
       }
 
-      this.objectStoreRest.createObject(query)
+      this.objectStoreRest.createObject(query, true)
```

Comment:
Don't pass boolean directly. Should create a variable and pass that. This helps with maintainability.

Reply:
Should not change existing APIs for this implementation.